apps/src/lib/tools/jsinterpreter/JSInterpreter.js

```diff
@@ -40,6 +41,29 @@ function safeStepInterpreter(jsi) {
   }
 }
 
+/**
+ * Checks the code row number and library name to determine whether we are in
+ * the user's code
+ */
+function isInUserCode(userCodeRow, libraryName) {
+  return -1 !== userCodeRow && !libraryName;
+}
+
+/**
+ * Checks if a library name exists in the node
```

Comment:
Can you change these comments to be a little less what and a little more why/how? ie. it's pretty easy to tell from the name of this function that it checks whether the node is from a library, but a comment about like, what `node.loc` is and how `node.loc.source` relates to libraries would be more useful IMO.